src/main/java/org/rulez/demokracia/pdengine/Voteable.java

```diff
@@ -6,16 +6,22 @@
 public interface Voteable extends VoteInterface {
 
 	default CastVote addCastVote(final String proxyId, final List<RankedChoice> theVote) {
+		CastVote castVote = new CastVote(proxyId, theVote);
+		getVotesCast().add(castVote);
+		return castVote;
+	}
+	
+	default CastVote addCastVote(final String proxyId, final List<RankedChoice> theVote, final List<String> assurances) {
 		Iterator<CastVote> listIterator = getVotesCast().iterator();
 		while (listIterator.hasNext()) {
 			CastVote element = listIterator.next();
 
-			if (element.proxyId != null && element.proxyId.equals(proxyId)) {
+			if (element.proxyId.equals(proxyId)) {
```

Comment:
Are you sure, element.proxyId will never be null?

Reply:
this one was an already approved mutant.